Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -1493,8 +1493,8 @@ checkCabalVersion pkg =
     allModuleNamesAutogen = concatMap autogenModules (allBuildInfo pkg)
 
 displayRawDependency :: Dependency -> String
-displayRawDependency (Dependency pkg vr) =
-  display pkg ++ " " ++ display vr
+displayRawDependency (Dependency pkg vr cs) =
+  display pkg ++ " " ++ display vr ++ " " ++ (unwords . fmap display . Set.toList $ cs) --MAYBE maybe drop cs entirely
```

Comment:
I guess it depends on what context `displayRawDependency` is used in. It looks like it is a faithful rendering, so I would drop it (in fact, I might suggest putting it in braces to demarcate it more clearly).